spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -226,6 +225,8 @@ public void onMessage(ConsumerRecord<Integer, String> message) {
 		container1.stop();
 		container2.stop();
 
+		props.put("auto.offset.reset", "earliest");
```

Comment:
Agreed - I looked at it, but eclipse sent me to the scala version - I now see them.